lib/hammer_cli/output/definition.rb

```diff
@@ -12,10 +12,117 @@ def append(fields)
       @fields += fields
     end
 
+    def delete_fields!(by = {})
+      return clear if by.empty?
+      return deep_delete!(@fields, by) if by[:deep_delete]
+      ids, labels = adjust_options(by)
+      @fields.delete_if do |f|
+        label = f.label
+        label = f.label.downcase unless label.nil?
+        ids.include?(f.id) || labels.include?(label)
+      end
+    end
+
+    def select_fields!(by = {})
```

Comment:
I'm not sure if there's some real-world use-case for this method. I can't imagine that somebody would want to keep just some fields and remove the rest.

It also doesn't work as expected. When I tested it on host info
```ruby
HammerCLIForeman::Host::InfoCommand.output_definition.select_fields!(:labels => ['Id'], :deep_select => true)
```
the info command then did contain only two "Id" fields, but none of them displayed any data.

I think it's fine to simple remove this method.